draft-twi-reference-architecture.md

```diff
@@ -217,7 +217,10 @@ Executing the Workload inside a Trusted Execution Environment can lower the risk
 
 # Security Considerations
 
-<cref>TODO</cref>
+Maintaining security guarantees of Confidential Computing within the WIMSE reference architecture calls for adding the extensions specified in this draft.
+Specifically, strong binding between the credential and the hosting enviroment, utilizing proof-of-possession of a workload credential backed by a key that is safeguarded against disclosure via data-in-use protections offered by Confidential Computing is a must-have.
```

Comment:
```suggestion
Specifically, strong binding between the credential and the underlying hardware platform, utilizing proof-of-possession of a workload credential backed by a key that is safeguarded against disclosure via data-in-use protections offered by Confidential Computing is a must-have.
```

Reply:
let us rename it to specific terms used in CC, as hosting environment links to host or OS software, which is not part of the CC TCB..